src/open_drive_roadways.py

```diff
@@ -5,10 +5,11 @@ def __init__(self):
         self.junctions = {}
 
 class Header:
-    def __init__(self, revMajor = 0, revMinor = 0, version = "",
+    def __init__(self, revMajor = 0, revMinor = 0, name = "", version = "",
             date = "", north = 0, south = 0, east = 0, west = 0):
         self.revMajor = int(revMajor)
         self.revMinor = int(revMinor)
+        self.name = name
```

Comment:
Name is not required in the header - or at least I assume so, since one of the OpenDrive files did not have that attribute. I think we should have self.name, but not have it as default? I can double check this as well

Reply:
From what I can tell, all the files we have parse this correctly. I think the variable has been there, just always name="". I'll double check and verify the Standard doesn't require it in all files.

Reply:
Ok great! Can you run the tests with main.py with both the `CarlaExs` folder and the `OpenDriveExs` folder to verify that this code doesn't throw any errors with any of the files? Thanks!

Reply:
Would be nice if the standard also told you whether attributes were required/optional. It doesn't say, but is part of the header according to Table 5 in the 1.6 standard. I tested on both folders of test files and everything seems okay. So I think it's safe for now, but maybe the next update we just have the parser check for each attribute before using them?

Reply:
Even more annoying is the tables in the 1.5 standard do have optional as a column.

@PeggyYuchunWang Do you think we should update the parser to check only the variables standard 1.5 suggests is optional in our next round of updates?

Reply:
Yeah, that would probably be best! I'll go ahead and merge it now then